etc/kapacitor/kapacitor.conf

```diff
@@ -321,7 +321,7 @@ default-retention-policy = ""
   enabled = false
   # The HipChat API URL. Replace subdomain with your
   # HipChat subdomain.
-  url = "https://subdomain.hipchat.com/v2/room"
+  url = ""
```

Comment:
Yeah, that should not have been there as it is not a valid configuration value. Instead of removing it can we leave it commented out so people can see the structure the URL they need to supply?